x-pack/plugins/alerting/common/routes/backfill/apis/schedule/schemas/v1.ts

```diff
@@ -15,6 +15,7 @@ export const scheduleBodySchema = schema.arrayOf(
       rule_id: schema.string(),
       start: schema.string(),
       end: schema.maybe(schema.string()),
+      run_actions: schema.maybe(schema.boolean({ defaultValue: true })),
```

Comment:
Should this be optional? Already has a default value.

Reply:
there are existing UI places that call this API so I think it has to be optional. I can remove the defaultValue if that makes more sense.